docs/ai-assistant-setup.md

```
@@ -0,0 +1,34 @@
+This document is targeted at NovaCrate users that want to make use of the integrated AI Assistant.
+
+# Opening the AI Assistant within NovaCrate
+
+The AI Assistant can be opened and closed by clicking the "sparkle" icon in the top right of the editor.
+
+![](./figures/ai-assistant-setup/find-ai-assistant.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)